experimental/devices/mcp9808/mcp9808.go

```diff
@@ -383,10 +383,10 @@ var (
 )
 
 func alertBitsToTemperature(b uint16) physic.Temperature {
-	b = (b >> 2) & 0x07FF
-	t := physic.Temperature(b&0x03FF) * 250 * physic.MilliKelvin
+	b = (b >> 2) & 0x07ff
```

Comment:
I think (?) the shift is not really needed, because one could do:
```
t := physic.Temperature(b&0x0fff) * 62500 * physic.MicroKelvin
if b&0x1000 != 0 {
  t -= 256 * physic.Celsius
}
return t + physic.ZeroCelsius
```

I find it a bit easier to read too but I'm not 100% sure the above is correct.

Reply:
Any strong opinion about this?